sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -44,8 +44,8 @@ class ContainerProxy(object):
     """An interface to interact with a specific DB Container.
 
     This class should not be instantiated directly. Instead, use the
-    :func:`DatabaseProxy.get_container_client` method to get an existing
-    container, or the :func:`Database.create_container` method to create a
+    :func:`~azure.cosmos.database.DatabaseProxy.get_container_client` method to get an existing
+    container, or the :func:`~azure.cosmos.database.DatabaseProxy.create_container` method to create a
```

Comment:
Do these also need to be updated in your async APIs?

Reply:
thanks for calling it out!